New process manual (#71)
aau-giraf/wiki: process_manual/introduction.md

# Introduction
The following will describe the process used in the 2019 project of the Giraf project.
It was updated at the very end of the project, to include all adaptations of processes, to help the year of 2020 get a headstart on the project.

> Most decisions will be described to explain the circumstances and arguments that made it the right decision at the time.

Reviewer: I don't know if it is appropriate to say "right"

We recommend that the year of 2020 try to replicate this process, and change it to match inevitable changes to your circumstances.
Best of luck.

## How to read this guide
This guide is intended to be read partly as a cover-to-cover guide of the process that we developed during the semester.
The primary focus, however, is for it to be a guide that you can go and look at, every time you are in doubt about the meaning of something.

> Are you a process group member you will probably benefit from reading it almost cover-to-cover or keeping it very close to you in the first few weeks.

Reviewer: ```suggestion
If you are a process group member, you will probably benefit from reading it almost cover-to-cover or keeping it very close to you in the first few weeks.
```

> Are you part of a development group, you will benefit greatly from the last sections, that describes how the code lifecycle works.

Reviewer: ```suggestion
If you are part of a development group, you will benefit greatly from the last sections that describe how the code lifecycle works.
```

Other than that, you might be able to use the rest of the sections for reference in your report.

## Introduction for the next process group
You have been chosen as the process group of this semester. It might be against your will, but nonetheless it's an important job that needs to be done.

> If this is at the very start of the semester we recommend that you read the following sections:

Reviewer: ```suggestion
If this is at the very beginning of the semester, we recommend that you read the following sections:
```

1. [General development in Process group information](process_group_information.md#general-development)
2. [Project meetings](project_meetings.md#project-meetings)
   1. Especially sprint intro and cross-group standups
3. [How to handle roles in Process group information](process_group_information.md#how-to-handle-roles)
4. [Changing the process](changing_the_process.md)

> Your responsibility is to define how project groups do everything: from writing proper code, to meetings, and releasing the code.

Reviewer: ```suggestion
Your responsibility is to define how project groups do everything: from writing proper code to meetings and releasing the code.
```
There will be no commas

Just remember: especially in the beginning it's a good idea to just take quick decisions and correct errors later. 30-40 people wait just for you to get started, so any plan is better than no plan.

> We do of course recommend our plan. That's still better than no plan.

Reviewer: Nice

Author: Thank you

To avoid micromanaging we don't care for how groups do things internally.
We only care about things that affect other groups as well.

> So if one groups wants to work hardcore SCRUM, and another don't want any formal process, let them.

Reviewer: ```suggestion
So if one groups wants to work hardcore Scrum, and another don't want any formal process, let them.
```
